### ISSUE TITLE
Add datasets

### ISSUE THREAD

**Issue by bevingtona** [bcgov/watershedBC#1]
- Soils https://cdnsciencepub.com/doi/full/10.1139/cjss-2021-0090
- Climate https://climatebc.ca/ https://daymet.ornl.gov/ https://github.com/bcgov/climr 
- BEC https://catalogue.data.gov.bc.ca/dataset/bec-zones-generalized-1-2m- 
- Forest species https://catalogue.data.gov.bc.ca/dataset/vri-2022-forest-vegetation-composite-polygons 
- Hydrologic recovery
- Riparian
- Hydrology https://www.canada.ca/en/environment-climate-change/services/water-overview/quantity/monitoring/survey.html

**Comment by bevingtona** [comment]
Very nice! I was not aware of this dataset. Sounds like it would be a good idea to change the datasource for the app +1.  The app currently uses a hard copy of the DRA. There are many limitations to road datasets in BC. Like construction date.. So very hard to use in a timeseries analysis. The roads don't plot up on the leaflet map automatically in the app. You you have to turn on the layer. 

![image](https://github.com/bcgov/watershedBC/assets/9651129/f37b9137-c744-49f9-ac72-0a40e1767999)

**Comment by Jhydromet** [comment]
Lemme know if I'm spamming your project here, but if you haven't encountered yet, there is great food for thought from [this cumulative effects report](https://www2.gov.bc.ca/assets/gov/environment/natural-resource-stewardship/cumulative-effects/protocols/cef-aquatic-ecosystems-protocol-dec2020.pdf)

Another suggested data set from this is [the human disturbance layer](https://catalogue.data.gov.bc.ca/dataset/ce-disturbance-2021) - includes seismic/transmission/rail lines as well

**Comment by bevingtona** [comment]
I agree, adding in the cummulative effects workflow for aquatic ecosystems would be a great benefit. We have the WHOPR in the Omineca, which is very similar, and I haven't gotten around to scripting it in a fast way that can be done on the fly. Largely because it depends on `VRI` which is rather slow. I do have the `VRI` duplicated in PostGIS, but it's still too slow. I might need to rasterize it! The human disturbance layer is great, too. I wonder about the data recency? I'm hoping watershedBC will be self-updating to the latest data in the catalogue for each of the inputs and not depend on another group to update layers.  

I'm thinking of trying to make a flowchart of data inputs required for `watershedBC` ... Need some sort of high level planning meeting!

**Comment by bevingtona** [comment]
Agriculture and urban data from 2000 : 2020 [data](https://open.canada.ca/data/en/dataset/fa84a70f-03ad-4946-b0f8-a3b481dd5248)

**Comment by bevingtona** [comment]
> Lemme know if I'm spamming your project here, but if you haven't encountered yet, there is great food for thought from [this cumulative effects report](https://www2.gov.bc.ca/assets/gov/environment/natural-resource-stewardship/cumulative-effects/protocols/cef-aquatic-ecosystems-protocol-dec2020.pdf)
> 
> Another suggested data set from this is [the human disturbance layer](https://catalogue.data.gov.bc.ca/dataset/ce-disturbance-2021) - includes seismic/transmission/rail lines as well

@Jhydromet  I added the CEF Human Disturbance data to the app. It's nice ! I'm very curious what the update schedule will be. It's already 2 years out of date.. but nice to have so many datasets consolidated. 

![image](https://github.com/bcgov/watershedBC/assets/9651129/30ba31c5-bb13-456f-982e-a17cc14e4a2b)

**Comment by Jhydromet** [comment]
Just heard some context on the CEF data update frequency - They are aiming for annual or biannually, but apparently these datasets are so coarse that they don't necessarily have the resolution/quality that updates really impact the values significantly...

Apparently they are invested in improving these datasets but it might be a long haul.. Sounds like its safest to treat these as static datasets for the time being

**Comment by bevingtona** [comment]
Sounds interesting, I think the point here is to have a framework that can easily include new datasets and updates. For now, this is probably OK, and we can work on improving the visualization, and even have a little inset map of the CEF data?
